sweetest/sweetest-api/src/main/java/com/mysugr/sweetest/framework/base/BaseJUnitTest.kt

```diff
@@ -2,29 +2,32 @@ package com.mysugr.sweetest.framework.base
 
 import com.mysugr.sweetest.framework.build.TestBuilder
 import com.mysugr.sweetest.framework.configuration.ModuleTestingConfiguration
+import com.mysugr.sweetest.framework.context.TestContext
+import com.mysugr.sweetest.usecases.finishWorkflow
+import com.mysugr.sweetest.usecases.proceedWorkflow
 import org.junit.After
 import org.junit.Before
 
 abstract class BaseJUnitTest @Deprecated(
     "No module configuration needed anymore.",
-    ReplaceWith("BaseJUnitTest()", imports = ["com.mysugr.sweetest.framework.base.BaseJUnitTest"])
```

Comment:
does the deprecation message work in terms of auto-replacing? for me it doesn't... @ChristopherKoellner

Reply:
Ja, zum Beispiel wie: 

![image](https://user-images.githubusercontent.com/56545896/100469650-f36be080-30d6-11eb-8390-baa09e503c80.png)

Reply:
Cool! I'm now curious why it doesn't for me... Does it correctly replace the right portion of the expression?

But I think I can remove the `imports = ["com.mysugr.sweetest.framework.base.BaseJUnitTest"` part of the `ReplaceWith` definition...

Reply:
Why would you remove that? 
It adds the import if there is none and does not add it if there is already one.